webapp/sources/rudder/rudder-rest/src/main/scala/com/normation/rudder/rest/EndpointsDefinition.scala

```diff
@@ -221,41 +243,49 @@ object GroupApi       extends Enum[GroupApi] with ApiModuleProvider[GroupApi] {
     val z: Int = implicitly[Line].value
     val description    = "List all groups with their information"
     val (action, path) = GET / "groups"
+    val authz: List[AuthorizationType] = AuthorizationType.Group.Read :: Nil
   }
   case object CreateGroup              extends GroupApi with GeneralApi with ZeroParam with StartsAtVersion2 with SortIndex {
     val z: Int = implicitly[Line].value
     val description    = "Create a new group"
     val (action, path) = PUT / "groups"
+    val authz: List[AuthorizationType] = AuthorizationType.Group.Write :: Nil
   }
   case object GetGroupTree             extends GroupApi with GeneralApi with ZeroParam with StartsAtVersion6 with SortIndex {
     val z: Int = implicitly[Line].value
     val description    = "List all group categories and group in a tree format"
     val (action, path) = GET / "groups" / "tree"
+    val authz: List[AuthorizationType] = AuthorizationType.Group.Read :: Nil
   }
   case object GroupDetails             extends GroupApi with GeneralApi with OneParam with StartsAtVersion2 with SortIndex  {
     val z: Int = implicitly[Line].value
     val description    = "Get information about the given group"
     val (action, path) = GET / "groups" / "{id}"
+    val authz: List[AuthorizationType] = AuthorizationType.Group.Read :: Nil
   }
   case object DeleteGroup              extends GroupApi with GeneralApi with OneParam with StartsAtVersion2 with SortIndex  {
     val z: Int = implicitly[Line].value
     val description    = "Delete given group"
     val (action, path) = DELETE / "groups" / "{id}"
+    val authz: List[AuthorizationType] = AuthorizationType.Group.Write :: Nil
   }
   case object UpdateGroup              extends GroupApi with GeneralApi with OneParam with StartsAtVersion2 with SortIndex  {
     val z: Int = implicitly[Line].value
     val description    = "Update given group"
     val (action, path) = POST / "groups" / "{id}"
+    val authz: List[AuthorizationType] = AuthorizationType.Group.Write :: AuthorizationType.Group.Edit :: Nil
   }
   case object ReloadGroup              extends GroupApi with GeneralApi with OneParam with StartsAtVersion2 with SortIndex  {
     val z: Int = implicitly[Line].value
     val description    = "Update given dynamic group node list"
     val (action, path) = GET / "groups" / "{id}" / "reload"
+    val authz: List[AuthorizationType] = AuthorizationType.Group.Read :: Nil
   }
```

Comment:
It could change the list of nodes in the group, so is it safe with `Read` instead of `Write + Edit` ?